config/routes.rb

```diff
@@ -3,4 +3,10 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  resources :scientists, only: [:show] do
+    resources :scientist_experiments, only: [:destroy], as: "scientist_experiments"
+  end
```

Comment:
Good fix! Without the `do` / `end` Rails doesn't know there is a relationship because there is no nesting, Rails sees it as 2 separate `resources` lines and one just happens to be indented.